.storybook/themes/commerce.scss

```
@@ -881,8 +881,8 @@
 					grid-template-rows: 1fr,
 					gap: var(--global-spacing-5),
 					margin: auto,
-					max-width: calc(var(--content-max-width) * 1px),
-					width: var(--content-scale-width),
+					max-width: 100%,
```

the max width can remain here since the mobile will never meet it and it's a sane default:
```suggestion
					max-width: calc(var(--content-max-width) * 1px),
```

---

@nschubach  thanks. Fixed this.